backend/controllers/projects.js

```diff
@@ -19,7 +19,7 @@ import Tool from '../models/tool.js';
 //basic CRUD functions:
 export const getAllProjects = async (req, res) => {
   try {
-    const projects = await Project.find().populate([{path: 'team_members', model: User}]); // .populate({path: "interested_applicants",model: User,  });
+    const projects = await Project.find().populate([{path: 'team_members', model: User}, {path: 'tools', model: Tool}, {path: 'interested_applicants', model: User}]); // .populate({path: "interested_applicants",model: User,  });
```

Comment:
create a ticket on jira, assign it to "back-end refactoring" epic (create it if it doesnt exist), and fill in title/description around the discussion of optimizing endpoints so that the application only holds information it needs. this will help improve memory-efficiency in the long run